packages/ripple-nuxt-tide/lib/middleware/banner-image.js

```diff
@@ -0,0 +1,26 @@
+
+export default async ({ app }, { tidePage }) => {
+  if (tidePage.field_landing_page_hero_image && tidePage.field_landing_page_hero_image.field_media_image) {
+    const mediaImage = tidePage.field_landing_page_hero_image.field_media_image
+    let focalPoint = null
+    try {
+      const cropResponse = await app.$tide.get(`crop/focal_point?filter%5Bentity_id%5D=${mediaImage.drupal_internal__fid}`)
```

Comment:
This will be moved to `field_media_image`, remove this additional call when that is implemented